src/examples/table/TableDraggable.tsx

```diff
@@ -0,0 +1,224 @@
+  const result = Array.from(list);
+  const [removed] = result.splice(startIndex, 1);
+
+  result.splice(endIndex, 0, removed);
+
+  return result;
+};
+
+const Example = () => {
+  const [items, setItems] = useState(generateItems(10));
+  const [recentDragId, setRecentDragId] = useState<string>();
+
+  useEffect(() => {
+    if (recentDragId) {
+      const draggable = document.getElementById(recentDragId);
+
+      draggable && draggable.focus();
+    }
+  }, [recentDragId]);
+
+  const onDragEnd = useCallback(
+    (result: DropResult) => {
+      if (!result.destination) {
+        return;
+      }
+
+      const newItems = reorderItems(items, result.source.index, result.destination.index);
+
+      setItems(newItems);
+      setRecentDragId(result.draggableId);
+    },
+    [items]
+  );
+
+  return (
+    <DragDropContext onDragEnd={onDragEnd}>
+      <Table>
+        <StyledCaption>Your Unsolved Tickets</StyledCaption>
+        <Head>
+          <HeaderRow>
+            <HeaderCell isMinimum />
+            <HeaderCell>Subject</HeaderCell>
+            <HeaderCell>Requester</HeaderCell>
+            <HeaderCell>Requested</HeaderCell>
+            <HeaderCell>Type</HeaderCell>
+          </HeaderRow>
+        </Head>
+        <Droppable droppableId="droppable">
+          {(droppableProps, droppableSnapshot) => {
+            return (
+              <Body ref={droppableProps.innerRef}>
+                {items.map((item, index) => (
+                  <Draggable key={item.id} draggableId={item.id} index={index}>
+                    {(provided, snapshot) => (
+                      <DraggableRow
+                        ref={provided.innerRef}
+                        isDraggingOver={droppableSnapshot.isDraggingOver}
+                        isHovered={snapshot.isDragging}
+                        isFocused={
+                          droppableSnapshot.isDraggingOver ? snapshot.isDragging : undefined
+                        }
+                        {...provided.draggableProps.style}
+                        {...provided.draggableProps}
+                      >
+                        <DraggableCell isDragOccurring={snapshot.isDragging}>
+                          <DraggableContainer id={item.id} {...provided.dragHandleProps}>
+                            <GripIcon />
+                          </DraggableContainer>
+                        </DraggableCell>
+                        <DraggableCell isDragOccurring={snapshot.isDragging}>
+                          {item.content}
+                        </DraggableCell>
+                        <DraggableCell isDragOccurring={snapshot.isDragging}>
+                          John Smith
+                        </DraggableCell>
+                        <DraggableCell isDragOccurring={snapshot.isDragging}>
+                          15 minutes ago
+                        </DraggableCell>
+                        <DraggableCell isDragOccurring={snapshot.isDragging}>Ticket</DraggableCell>
```

Comment:
Recommend getting help obtaining row/column content that is coherent with the existing site.

Reply:
Updated to match existing table data.